### ISSUE TITLE
Application of phase regression to multi-echo BOLD

### ISSUE THREAD

**Issue by tsalo** [nipreps/fmripost-phase#12]
@cesarcaballerogaudes @eurunuela @idevicente have any of you tried applying phase regression to multi-echo data? Asking you three because of your connections to https://github.com/idevicente/Phase-based_denoising_pipeline.

Stems from #9 and #2.

**Comment by eurunuela** [comment]
We used single-echo data for that work. I cannot remember if Iñigo tried it on multi-echo data though.

**Comment by CesarCaballeroGaudes** [comment]
We haven't tried yet, but we have collected new multi-echo data with both magnitude and phase in several projects.

**Comment by tsalo** [comment]
The easiest thing would be to just use the echo closest to 30 ms I guess, but I wonder if there would be any benefit to either averaging the complex-valued data across echoes before running the phase regression pipeline or running the pipeline on each echo separately and then averaging the denoised magnitude data afterward.

**Comment by CesarCaballeroGaudes** [comment]
No, we haven't done any progress on phase regression, so far only using the phase for multi-echo thermal denoising
